week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,17 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+
+function tidyUpString(strArr) {
+  let i = 0;
```

Comment:
This works, but I'd expect the `let` to be in the `for` loop to limit its scope, i.e.
```javascript
for (let i= 0; i < strArr.length; i++) {
```
is there a reason you're declaring it outside?

Reply:
i actually did it this way at first but during the call my colleagues seem to be unaware of possibility to declare the variable inside a loop and i later changed it.